src/site/stages/build/plugins/download-drupal-assets.js

```diff
@@ -16,7 +16,32 @@ async function downloadFile(
   everythingDownloaded,
 ) {
   const asset = assetsToDownload.shift();
-  const response = await client.proxyFetch(asset.src);
+  const fileOutputPath = path.join(
+    options.cacheDirectory,
+    'drupal/downloads',
+    asset.dest,
+  );
+
+  let response;
+  let retries = 3;
+  while (retries--) {
+    try {
+      // eslint-disable-next-line no-await-in-loop
+      response = await client.proxyFetch(asset.src);
+      break;
+    } catch (e) {
+      if (retries > 0) {
+        // eslint-disable-next-line no-console
+        console.error(
+          `Error while fetching ${
+            asset.src
+          }. ${e} Retries remaining: ${retries}`,
+        );
+      } else {
+        throw e;
```

Comment:
This code looks good but it would be cool if we could test this reliably, because it's confusing in the context that several assets are downloaded in parallel and they may all run into this at the same time. Can we add a pause here maybe? Just a simple 

```js
await new Promise(resolve => setTimeout(500, resolve));
```

To reduce strain on the proxy before trying again